react/app-admin/components/OverviewPage/OverviewPage.tsx

```diff
@@ -1,25 +1,21 @@
 import * as React from 'react';
 import { Message, Loader, Table, Label } from 'semantic-ui-react';
 import { ActionMessage } from 'tmw-admin/components/ActionMessage';
-import { VisitorsChart } from 'tmw-admin/components/OverviewPage'
+import { VisitorsChart } from 'tmw-admin/components/OverviewPage';
 import { PageHeader } from 'tmw-admin/components/PageHeader';
 import { serializeLogsFromAPI, serializeVisitorStatsFromAPI } from 'tmw-admin/utils/api-serialize';
 import { ajaxGet, ajaxPost } from 'tmw-common/utils/ajax';
-import { getApiDateFormat } from '../../../common/utils/date';
+import { getApiDateFormat, getTimeFromApiDate } from '../../../common/utils/date';
```

Comment:
I would put the absolute path here to be consistent with the rest of the app (+ it's easier if we move the file later)

Reply:
You're right, I just did it ! 👍